tests/lambdas/leverDev.test.js

```diff
@@ -0,0 +1,79 @@
+const lambda = require('../../src/functions/lever')
+const got = require('got')
+const url = require('url')
+
+jest.mock('got')
+jest.mock('url')
+
+describe('different data, not production', () => {
+  let response
+
+  beforeAll(async () => {
```

Comment:
Usually `beforeAll` blocks are used to set up/take down/reset mocks and other functionalities that the test needs to function. It is possible to but not usually used to perform the execution of the function at hand.